config/initializers/cookies_serializer.rb

```diff
@@ -2,4 +2,5 @@
 
 # Specify a serializer for the signed and encrypted cookie jars.
 # Valid options are :json, :marshal, and :hybrid.
-Rails.application.config.action_dispatch.cookies_serializer = :marshal
+# TODO: change this to :json on the next rails update
+Rails.application.config.action_dispatch.cookies_serializer = :hybrid
```

Comment:
I tried to change to cookie serializer once and it caused thousands of cookie deserialisation errors. I am pretty sure I also used hybrid which is supposed to be the upgrade path, right? But it's been a while. Maybe thinks have gotten better? ;)